packages/okta-react-native/README.md

```diff
@@ -50,145 +40,218 @@ After you have created the application there are two more values you will need t
+4. Under `URL Schemes`, add your scheme to `Item 0` -->
+
+### Android Setup
+
+#### Install Okta Open ID Connect Android
+This library depends on the native [Okta OIDC Android](https://github.com/okta/okta-oidc-android) library. You have to add this library through Gradle. Follow the following steps:
 
-### Testing on Android Devices
+1. Add this line to `android/build.gradle`, under `allprojects` -> `repositories`.
 
-There is a [known issue](https://github.com/okta/okta-sdk-appauth-android/issues/8) when redirecting back to a URI scheme from the browser via Chrome Custom Tabs. This is due to Chrome **not supporting** JavaScript initiated redirects back to native applications.
+		maven {
+			url  "https://dl.bintray.com/okta/com.okta.android"
+			}
+		
+2. Make sure your `minSdkVersion` is `19` in `android/build.gradle`.
 
-To handle this, please refer to the workaround recorded in [this issue](https://github.com/okta/okta-sdk-appauth-android/issues/8).
+#### Add redirect scheme
+
+Defining a redirect scheme to capture the authorization redirect. In `android/app/build.gradle`, under `android` -> `defaultConfig`, add:
+```
+manifestPlaceholders = [
+  appAuthRedirectScheme: '{YOUR_REDIRECT_SCHEME}'
+]
+```
 
 ## Usage
 
 You will need the values from the OIDC client that you created in the previous step to instantiate the client. You will also need to know your Okta Org URL, which you can see on the home page of the Okta Developer console.
 
-In your application's controller, create a new instance of the `TokenClient`:
+Before calling any other method, it is important that you call `createConfig` to set up the configuration properly on the native modules.
+
+Importing methods would follow this pattern: 
 
 ```javascript
-import TokenClient from '@okta/okta-react-native';
-
-const tokenClient = new TokenClient({
-  issuer: 'https://{yourOktaDomain}.com/oauth2/default',
-  client_id: '{clientId}',
-  redirect_uri: __DEV__ ?
-    'exp://localhost:19000/+expo-auth-session' :
-    'com.oktapreview.{yourOrg}:/+expo-auth-session'
-});
+import { createConfig, signIn, signOut, getAccessToken } from '@okta/okta-react-native';
 ```
-**Note:** *redirect_uri will need to match the ones you configured in Okta*
 
-There are additional configuration options you can provide for specialized use cases:
+### `createConfig`
 
-* `authorization_endpoint` - override the authorization endpoint from the well-known endpoint
-* `storageKey` - Unique key used to store/retrieve secure data
-* `keychainService` - See [Expo - keychainService](https://docs.expo.io/versions/latest/sdk/securestore.html#keychainservice-string-)
-* `keychainAccessible` - (iOS only) Specify when the stored item is accessible. See [Expo - keychainAccessible](https://docs.expo.io/versions/latest/sdk/securestore.html#keychainaccessible-enum-)
+This method will create a configured client on the native modules.
 
-### `signInWithRedirect`
+```javascript
+await createConfig(
+	'<YOUR_CLIENT_ID>', 
```

Comment:
elsewhere we have a fairly standardized format for these using `{}` (e.g. `{clientId}`).  Is there a mobile standard to use `<>`?

Reply:
i'll make sure to change into the standardized format